recipes/clinfo/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Hi @isuruf , should we have this shebang?
Because ``conda-build`` already handles it, right?

Reply:
Yes, but it doesn't hurt to have and helps in `conda debug`